docs/en/sql-reference/statements/alter/role.md

```diff
@@ -10,7 +10,7 @@ Changes roles.
 Syntax:
 
 ``` sql
-ALTER ROLE [IF EXISTS] name [ON CLUSTER cluster_name]
-    [RENAME TO new_name]
+ALTER {ROLE | ROLES} [IF EXISTS] name1 [ON CLUSTER cluster_name1] [RENAME TO new_name1] 
```

Comment:
There is no command `ALTER ROLES`. Only `ALTER ROLE` exists, it just allows using multiple roles, kind of

```
ALTER ROLE role1, role2, ...
```

The same thing for other ALTER/CREATE commands.

Reply:
@vitlibar Fixed that